### ISSUE TITLE
Uncaught Error: Call to a member function attempt_reset() on null

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-bwtrace#99]
[01-Mar-2021 20:37:33 UTC] PHP Fatal error:  Uncaught Error: Call to a member function attempt_reset() on null in /home/cookie/public_html/wp-content/plugins/oik-bwtrace/includes/class-BW-trace-controller.php:493
```
Stack trace:
#0 /home/cookie/public_html/wp-content/plugins/oik-bwtrace/includes/oik-actions.php(754): BW_trace_controller->purge_trace_file_if_no_errors()
#1 /home/cookie/public_html/wp-includes/class-wp-hook.php(287): bw_trace_purge_if_no_errors('')
#2 /home/cookie/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#3 /home/cookie/public_html/wp-includes/plugin.php(484): WP_Hook->do_action(Array)
#4 /home/cookie/public_html/wp-includes/load.php(1052): do_action('shutdown')
#5 [internal function]: shutdown_action_hook()
#6 {main}
  thrown in /home/cookie/public_html/wp-content/plugins/oik-bwtrace/includes/class-BW-trace-controller.php on line 493
```

**Comment by bobbingwide** [comment]
This has happened a few times. Most recently I got the problem while running `wp help core` in `s.b/cwiccer`

**Comment by bobbingwide** [comment]
This has now been released in v3.2.1. So closing.
